examples/custom_msg.py

```diff
@@ -43,10 +43,10 @@ class CustomModbusResponse(ModbusPDU):
     function_code = 55
     rtu_byte_count_pos = 2
 
-    def __init__(self, values=None, device_id=1, transaction=0):
+    def __init__(self, values: list[int] | None = None, device_id=1, transaction=0):
         """Initialize."""
         super().__init__(dev_id=device_id, transaction_id=transaction)
-        self.values = values or []
+        self.values: list[int] = values or []
```

Comment:
self.values should automatically get list[int] because the expression removes the None version.

Reply:
Bah, pylint also complains: https://pylint.pycqa.org/en/latest/user_guide/messages/warning/dangerous-default-value.html